src/switch_core_media.c

```diff
@@ -5108,7 +5108,7 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 
 			if (switch_channel_var_false(smh->session->channel, "telnyx_disable_rtcp") || !switch_channel_get_variable(smh->session->channel, "telnyx_disable_rtcp")) {
 				if (remote_rtcp_port) {
-					if (switch_channel_var_true(smh->session->channel, "force_rtcp_passthru")) {
+					if (switch_channel_var_true(smh->session->channel, "force_rtcp_passthru") || smh->mparams->force_rtcp_passthru) {
```

Comment:
If we set `force_rtcp_passthru` in the dialplan as false then that also takes precedence over the profile variables. Please handle that situation including the rest of the new/existing vars

Reply:
I think I got it now. Please re-re-review again :)